mon-pix/translations/en.json

```diff
@@ -75,7 +75,7 @@
             "assessment": {
                 "title": "Begin your personalised test",
                 "action": "Begin",
-                "announcement": "Start your personalised test.<br /> Sign up or log in to the Pix platform and start your test.",
+                "announcement": "Start your personalised test.<br> Sign up or log in to the Pix platform and start your test.",
```

Comment:
Question : Quelle est la différence entre 
- <br /> => **<br>**
et
<br /> => **<b>** ?

Reply:
- `br`: permet un retour à la ligne
- `b` permet de mettre en gras (=`bold`)
- `<br>` et `<br />` sont tous les deux valides en HTML. Mais en XHTML, seul `<br />` sera valide car il faut le '/'

Une petite explication sur le screenshot ci-dessous, tiré de https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br
![Screenshot 2020-10-21 at 12 25 31](https://user-images.githubusercontent.com/11294487/96719525-c62a6500-13a9-11eb-961f-7be18dd03100.png)